runtime/src/invoke_dpe.rs

```diff
@@ -10,6 +10,14 @@ pub struct InvokeDpeCmd;
 impl InvokeDpeCmd {
     pub(crate) fn execute(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<MailboxResp> {
         if let Some(cmd) = InvokeDpeReq::read_from(cmd_args) {
+            // Validate data length
+            let Ok(in_len) = usize::try_from(cmd.data_size) else {
```

Comment:
nit: Caliptra firmware won't ever run on a processor that is less than 32-bit, so `cmd.data_size as usize` is safe.

Reply:
Done